crypto/rand/src/lib.rs

```diff
@@ -16,6 +16,9 @@ cfg_if! {
     if #[cfg(target_feature = "rdrand")] {
         mod rdrandrng;
         pub use rdrandrng::McRng;
+    } else if #[cfg(all(target_arch = "wasm32", target_os = "unknown"))] {
```

Comment:
This mimics the cfg check the underlying `getrandom` crate is doing: https://github.com/rust-random/getrandom/blob/master/src/lib.rs#L238

Reply:
disregard me comment, @eranrund  is correct. it looks like the os check is a way to ensure it's for `wasm32-unkown-unknown` since there is no cfg option for the target triple directly
```shell
➜  ~ rustc --print target-list | grep wasm32
wasm32-unknown-emscripten
wasm32-unknown-unknown
wasm32-wasi
➜  ~ rustc +nightly -Z unstable-options --print target-spec-json --target wasm32-unknown-unknown | jq .os
"unknown"
➜  ~ rustc +nightly -Z unstable-options --print target-spec-json --target wasm32-unknown-emscripten | jq .os
"emscripten"
➜  ~ rustc +nightly -Z unstable-options --print target-spec-json --target wasm32-wasi | jq .os
"wasi"
```